### ISSUE TITLE
block implicitly retains self

### ISSUE THREAD

**Issue by wtholliday** [ollix/MetalNanoVG#20]
I'm getting this warning:

```
nanovg_mtl.m:1387:33: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
      dispatch_semaphore_signal(_semaphore);
                                ^
                                self->
```

Easy to fix :)

**Comment by olliwang** [comment]
I couldn't reproduce the same warning on Xcode 11 beta 5. What Xcode version do you use?

**Comment by wtholliday** [comment]
@olliwang Version 10.3 (10G8)

**Comment by olliwang** [comment]
Thanks. Fixed in d3e0fc65180f6f2cb1a9.
